packages/zoe/src/contracts/types-ambient.js

```diff
@@ -4,11 +4,15 @@
  * @property {() => Amount} getAvailableItems
  *
  * @typedef {object} SellItemsCreatorOnly
- * @property {() => Promise<import('../types-index.js').Invitation>} makeBuyerInvitation
+ * @property {() => Promise<Invitation>} makeBuyerInvitation
  *
  * @typedef {SellItemsPublicFacet & SellItemsCreatorOnly} SellItemsCreatorFacet
  */
 
+/**
+ * @import {Invitation} from '../types-index.js';
```

Comment:
This might change the semantics of the file into a "module" vs a script, and break the ambient declarations below.